Podfile.lock

```diff
@@ -54,7 +54,7 @@ PODS:
   - Specta (1.0.7)
   - SVProgressHUD (2.2.5)
   - UIDeviceIdentifier (1.1.4)
-  - WordPressKit (3.1.1):
+  - WordPressKit (3.2.1):
```

Comment:
When I do pod update after checking the branch I see a warning:
[!] GTMOAuth2 has been deprecated in favor of GTMAppAuth
Is that expected?

Reply:
That's a great question! Yes that's supposed to happen. @jtreanor has been doing extensive work to get Authenticator up-to-date so it's compatible with CocoaPods version 1.6.1. So now we are free to update the `GoogleSignIn` pod (which is where that warning is originating from). I am not going to do that in this PR, but I can file a task so the warning will not get lost.